test/release/examples/primers/domains.chpl

```diff
@@ -30,49 +30,46 @@
 config var n = 10;
 
 var RD: domain(3) = {1..n, 1..n, 1..n};
-writeln(RD);
+writeln(RD); // {1..10, 1..10, 1..10}
 
 //
 // Rectangular domains have a set of methods that enable convenient
 // reuse of existing domains.
 //
 // The :proc:`~ChapelDomain.expand` method returns a new domain that is expanded or
-// contracted depending on the sign of the offset argument.
+// contracted depending on the sign of the offset argument.  The
+// expansion/contraction happens symmetrically to the lower and upper bound.
 //
-var RDbigger = RD.expand((1,1,1));
-writeln(RDbigger);
-var RDsmaller = RD.expand((-1,-1,-1));
-writeln(RDsmaller);
+var RDexp = RD.expand((1,2,-2));
+writeln(RDexp); // {0..11, -1..12, 3..8}
 
 //
-// The :proc:`~ChapelDomain.exterior` method returns a new domain that is the
-// exterior portion of the current domain.  A positive offset specifies that
-// the exterior should be taken from the high bound; a negative offset, the low
-// bound.
+// The :proc:`~ChapelDomain.exterior` method returns a new domain that is an
+// exterior portion of the current domain.  This is not the "border" of the
+// domain (which is not a rectangular domain), in particular the new domain
+// will share no indices with the original domain.  Inputs specify the width
+// of the new domain in the respective dimension.  A positive value specifies
+// that the exterior should be taken from the high bound; a negative offset,
+// the low bound.
 //
-var RDext_p = RD.exterior((1,1,1));
-writeln(RDext_p);
-var RDext_n = RD.exterior((-1,-1,-1));
-writeln(RDext_n);
+var RDext = RD.exterior((1,4,-4));
+writeln(RDext); // {11..11, 11..14, -3..0}
 
 //
 // The :proc:`~ChapelDomain.interior` method returns a new domain that is the
-// interior portion of the current domain.  The sign of the offset implies
+// interior portion of the current domain.  Inputs specify the width
+// of the new domain in the respective dimension.  The sign of the offset implies
 // using the high or low bound as in the exterior case.
 //
-var RDint_p = RD.interior((1,1,1));
-writeln(RDint_p);
-var RDint_n = RD.interior((-1,-1,-1));
-writeln(RDint_n);
+var RDint = RD.interior((2,-5,1));
+writeln(RDint); // {9..10, 1..5, 10..10}
 
 //
 // The :proc:`~ChapelDomain.translate` method returns a new domain that is the
 // current domain translated by the offset.
 //
-var RDtrans_p = RD.translate((1,1,1));
-writeln(RDtrans_p);
-var RDtrans_n = RD.translate((-1,-1,-1));
-writeln(RDtrans_n);
+var RDtrans = RD.translate((0,4,-4));
+writeln(RDtrans); // {1..10, 5..14, -3..6}
```

Comment:
It would be nice also to show and explain that a single offset can be applied to all dimensions in the above functions, for example:

```chpl
writeln(RD.translate(-3));  // {-2..7, -2..7, -2..7}
```

Reply:
Done!